ccswbs/hjckrrh
# Front End Performance Feature

This pull request includes the UG Optimization feature and its two dependencies: advagg and imagemagick. All three modules are disabled by default, so this can be merged without impacting anything.

## Test Procedure
1. Enable UG Optimization
2. Revert UG Optimization feature. This should only include one line related to Image Magick
3. Compare test site configuration for advagg and imagemagick against http://aoda-test.web.uoguelph.ca/ccs
4. Add `$conf['advagg_skip_404_check'] = TRUE;` to settings.php
5. Ensure Status Report doesn't have any errors
6. Open developer tools and check the console for any 404 errors related to requests for js or css files
